### DISCUSSION TITLE
Twitter thread about LinkFree

### DISCUSSION THREAD

**Discussion by iAnonymous3000** [EddieHubCommunity/support#7047] [General]
Hey everyone, I just created a Twitter thread about Linkfree and how it can help you showcase your work and projects all in one place. If you're interested in learning more, check it out at the link below! And if you find it helpful, please share it with your friends and followers to help spread the word about this great platform.

Link to Twitter thread: https://twitter.com/iAnonymous3000/status/1635314275337256960

**Comment by Vijaykv5** [comment]
Sure :)

**Comment by bkpecho** [comment]
Nice work brother @iAnonymous3000! Also, I followed you :) I'm @bkpecho 😄

**Comment by Pradumnasaraf** [comment]
Great work, @iAnonymous3000 🎉
